ts/Dashboard/Component/SharedComponentState.ts

```diff
@@ -230,13 +232,20 @@ Serializable<SharedComponentState, SharedComponentState.JSON> {
     }
 
     public setHoverPoint(
-        point?: SharedComponentState.PresentationHoverPointType,
-        eventDetail?: {}
+        point?: SharedComponentState.PresentationHoverPointType | HTMLElement,
+        eventDetail?: any
```

Comment:
`any` should be fixed, as an object with `isDataGrid` property is used. At least it should become `AnyRecord`.